### PR TITLE
💅Add Styled components

### PR DESCRIPTION
**Pull request by bwnstck**
# Description
Add globalStyles to Storybook
Fixes #26 

## Type of change
- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

**Comment by ncbecker** [comment]
lgtm 👍

File: client/src/globalStyles.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { createGlobalStyle } from "styled-components";
```
Comment (lmachens) [inline client/src/globalStyles.js:1]: You could improve your developer experience with https://styled-components.com/docs/tooling#babel-macro

```suggestion
import { createGlobalStyle } from "styled-components/macro";
```

File: client/src/App.js [hunk 2]
```diff
@@ -1,7 +1,13 @@
 import React from "react";
+import GlobalStyle from "./globalStyles";
```
Comment (lmachens) [inline client/src/App.js]: If you name other components in PascalCase, consider to rename the module to `GlobalStyle.js`. 
I renamed the file to `GlobalStyle` (singular).

**Review by lmachens — COMMENTED** [review]
gj

**Pull request merged**